pkg/maps/egressmap/egress.go

```diff
@@ -5,20 +5,18 @@ package egressmap
 
 const (
 	PolicyMapName = "cilium_egress_gw_policy_v4"
-
-	MaxPolicyEntries = 1 << 14
 )
 
 var (
 	EgressPolicyMap *egressPolicyMap
 )
 
 // InitEgressMaps initializes the egress policy map.
-func InitEgressMaps() error {
-	return initEgressPolicyMap(PolicyMapName, true)
+func InitEgressMaps(maxPolicyEntries int) error {
+	return initEgressPolicyMap(PolicyMapName, maxPolicyEntries, true)
 }
 
 // OpenEgressMaps initializes the egress policy map.
-func OpenEgressMaps() error {
-	return initEgressPolicyMap(PolicyMapName, false)
+func OpenEgressMaps(maxPolicyEntries int) error {
```

Comment:
I don't think we need to pass `maxPolicyEntries` to `OpenEgressMaps` (only needed for `InitEgressMaps`)

Reply:
oh thanks!  parameter `create` is false in `OpenEgressMaps`. so we actually don't need pass `maxPolicyEntries`